frontend/src/components/Interactions/Menu.vue

```diff
@@ -1,19 +1,90 @@
 <template>
-    <button
-        class="material-icons menu-button"
-        type="menu"
-        title="Toggle Menu"
-    >
-        more_vert
-    </button>
+    <div class="menu">
+        <button
+            class="material-icons menu-button"
+            type="menu"
+            title="Toggle Menu"
+            @click="toggleMenu"
+            @blur="blurHandler"
+        >
+            {{toggled ? 'not_interested' : 'more_vert'}}
+        </button>
+        <ol
+            ref="dropdown"
+            :class="['menu-items', {'drop-down-active': toggled}]"
```

Comment:
wouldn't just `v-show` be a more concise way of toggling the menu visibility?

Reply:
Hmmm. Maybe.